f600233d1b154d507b782f84604bcb12/update/sys_script_include_da81226197072110cf3cfd9fe153afee.xml

```diff
@@ -31,7 +31,7 @@ MarketResearchReport.prototype = {
      * @return {array} technique objects with technique_value/sequence properties
      */
     getTechniquesUsed: function(techniqueSysIds) {
-        const PROPERTY_NAMES = ['technique_value', 'sequence'];
+        const PROPERTY_NAMES = ['technique_value', 'technique_label', 'sequence'];
```

Comment:
Requests an additional column from the reference table `x_g_dis_atat_dapps_market_research_other_techniques`